resources/sample_vaults/Tasks-Demo/Styling/ASCII Tasks Table.md

```diff
@@ -79,7 +79,7 @@ To:
 - []] #task ASCII code 93 - `]` - box brackets or square brackets, closing bracket
 - [^] #task ASCII code 94 - `^` - Circumflex accent or Caret
 - [_] #task ASCII code 95 - `_` - underscore , understrike , underbar or low line
-- [`] #task ASCII code 96 - `backtick` - Grave accent
+- [`] #task ASCII code 96 -`backtick` - Grave accent
```

Comment:
This breaks the formatting...
I've suppressed this edit in a separate commit - 6ab4b219bd1a372f8ec56849dd12f68e640b880d